server/routes/journeys/record-review/participants-summary/view.njk

```diff
@@ -0,0 +1,118 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends "partials/layout.njk" %}
+
+{% set fullWidth = attendees.length %}
+{% set pageTitle = "Participant and contribution details - Record a CSIP review" %}
+
+{% block innerContent %}
+    {% include 'base/components/review-caption.njk' %}
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-7">Log participants and contributions to the review</h1>
+
+    {% for attendee in attendees %}
+        {{ govukSummaryList({
+            card: {
+                title: { text: "Participant: " + attendee.name },
+                actions: {
+                    items: [
+                        {
+                            href: 'delete-participant/' + attendee.attendeeUuid,
```

Comment:
attendees don't have uuid yet, we can only link to them via index

same with the urls below

Reply:
It's in the types for them though?

Reply:
probably when we `record-review` we use `/:index`,
later on when we `update-review` we will use `/:attendeeUuid`.

i think we did the same with contributory factors and interviewees and identified needs.

Reply:
If we have the uuid we may as well use it right? We talked about wanting to refactor the others to use it at some point.

Reply:
@bill-moj ? :)